src/document.rs

```diff
@@ -42,8 +43,15 @@ impl MetadataName {
 
 #[derive(Debug, Clone, Copy, PartialEq)]
 pub struct Location {
-    pub chapter: i32,
-    pub page: i32,
+    pub chapter: u32,
+    /// Index of the page inside the inside the [`chapter`](Location::chapter).
+    ///
+    /// See [`page_number`](Location::page_number) for the absolute page number.
+    pub page: u32,
```

Comment:
```suggestion
    pub page_in_chapter: u32,
```

What do you think?

Reply:
Sounds good. And keep `page_number`? Or call that `page`?